.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,85 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: "**Required Info:**"
+  - type: input
+    id: os
+    attributes:
+      label: "Operating System:"
+      description: |
+        Please try to be specific.
+        For linux please use the command `uname -a` from a terminal and copy paste its output here.
+        For windows, open a terminal (Win key + R and type `cmd`), type the command `ver` and press enter. Then copy paste the output here.
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: "ROS version or commit hash:"
+      description: |
+        **Examples:**  *humble*, *iron*, *jazzy*, ...
+    validations:
+      required: true
+  - type: input
+    id: dds
+    attributes:
+      label: "RMW implementation:"
+      description: |
+        **Examples:** *rmw_fastrtps_cpp*, *rmw_connextdds*, *rmw_cyclonedds_cpp*, ...
+        You can check the ROS Middleware (RMW) implementation with the command: `ros2 doctor --report`
+        Find the line starting with `middleware name` in the report.
+    validations:
+      required: true
+  - type: input
+    id: clientlib
+    attributes:
+      label: "Client library (if applicable):"
+      description: |
+        **Examples:** *rclcpp*, *rclpy*, ...
+        Client libraries are the APIs that allow users to implement their ROS 2 code.
+    validations:
+      required: false
+  - type: textarea
+    id: doctor
+    attributes:
+      label: "'ros2 doctor --report' output"
+      description: |
+        It can help us knowing the details of your ROS environment.
+        Please use the command `ros2 doctor --report` and copy paste it's output here.
+      render: Formatted
+    validations:
+      required: false
+  - type: textarea
+    id: repro
+    attributes:
+      label: "Steps to reproduce issue"
+      description: "How do you trigger this bug? Please walk us through it step by step."
```

Since this part is very important, I think we should be more specific here:

```suggestion
      description: |
        How do you trigger this bug? Please walk us through it step by step.
        Include all the commands you ran in the exact order you ran them so that anyone can reproduce the bug.
```